### PR TITLE
Internal: Editor V2 - Adapt styles [ED-9206]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: core/editor/config-providers/editor-common-assets.php [hunk 1]
```diff
@@ -113,4 +113,46 @@ public static function get_scripts() {
...
+				'version' => '1.12.0',
+			],
+			[
+				'handle' => 'pickr',
+				'src' => '{{ASSETS_URL}}lib/pickr/themes/monolith.min.css',
+				'version' => '1.5.0',
+			],
+			[
+				'handle' => 'elementor-editor',
+				'src' => '{{ASSETS_URL}}css/editor{{DIRECTION_SUFFIX}}{{SUFFIX}}.css',
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply core/editor/config-providers/editor-common-assets.php:145]: Done

File: core/editor/assets/css/editor-v2-overrides.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply core/editor/assets/css/editor-v2-overrides.css:1]: Done

File: core/editor/config-providers/editor-v2-config-provider.php [hunk 3]
```diff
@@ -25,31 +25,46 @@ public function get_scripts() {
...
+				],
 			]
 		);
 	}
 
-	public function get_loader_scripts() {
-		$deps = [
-			// Feature packages.
+	public function get_scripts_for_enqueue() {
+		return [
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply core/editor/config-providers/editor-v2-config-provider.php:43]: Done

File: core/editor/editor-loader.php [hunk 4]
```diff
@@ -74,21 +94,27 @@ public function print_root_template() {
 	 * Normalize script configs for enqueue and register methods.
 	 *
 	 * @param array $script_configs
+	 * @param string $type can be ['script', 'style']
 	 *
 	 * @return array
 	 */
-	private function normalize_script_configs( array $script_configs ) {
-		$default = [
+	private function normalize_script_configs( array $script_configs, $type ) {
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply core/editor/editor-loader.php:101]: Done

File: core/editor/editor-loader.php [hunk 5]
```diff
@@ -71,24 +91,30 @@ public function print_root_template() {
...
 			'version' => ELEMENTOR_VERSION,
 			'in_footer' => true,
-		];
+		], $additional_defaults );
 
 		$replacements = [
 			'{{ASSETS_URL}}' => ELEMENTOR_ASSETS_URL,
-			'{{SUFFIX}}' => ( Utils::is_script_debug() || Utils::is_elementor_tests() ) ? '' : '.min',
+			'{{MIN_SUFFIX}}' => ( Utils::is_script_debug() || Utils::is_elementor_tests() ) ? '' : '.min',
+			'{{DIR_SUFFIX}}' => is_rtl() ? '.rtl' : '',
```
Comment (StyleShit) [inline core/editor/editor-loader.php:117]: not sure about "dir". it usually means "directory"

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: core/editor/editor-loader.php (hunk 5 above)
Comment (Nevoss) [reply core/editor/editor-loader.php:117]: `dir: rtl`
min - minimum
dir - direction

if I make shortcut for min I should make shortcut for dir

**Review by StyleShit — APPROVED** [review]

**Review by rotemee — APPROVED** [review]

**Pull request merged**